packages/kerosene-test/src/cypress/support/commands.js

```diff
@@ -6,6 +6,6 @@ Cypress.Commands.add("visitStorybook", (path, storybookName, options = {}) => {
     options,
   );
 });
-Cypress.Commands.add("testId", testId => {
-  cy.get(`[data-test-id="${testId}"]`);
+Cypress.Commands.add("testId", (testId, options) => {
+  cy.get(`[data-test-id="${testId}"]`, options);
```

Comment:
Looks good, but do we need an option for the `data-testid` case?

Reply:
yeah we need to work out how to make it happen, ie configuring which test-id convention you're using.

theres probably a way, but i'm not sure how to connect cypress config to a command, maybe theres a way to read it?

Probably out of scope of this PR.

Reply:
Captured it here #67